keps/sig-auth/20190607-certificates-api.md

```diff
@@ -145,6 +171,20 @@ type CertificateSigningRequest struct {
 
 
 type CertificateSigningRequestSpec struct {
+  // requested signer for the request.  It is a scoped name in the form: `scope-hostname.com/name`.  
```

Comment:
`scope-hostname.com/name` is confusing given that all of the standard signers have an `.io` TLD. Say that the signeName must be a qualified name, assuming we already document what that means.